docs/source/yelpsoa_configs.rst

```diff
@@ -17,6 +17,9 @@ specify the following options:
     available free CPU, but is guaranteed to get the CPU shares specified.  For
     a more detailed read on how this works in practice, see the docs on `isolation <isolation.html>`_.
 
+    * ``cpu_burst_add``: Number of additional CPUs an instance may use while
```

Comment:
The indentation looks off here. Can you double check?

Reply:
The indentation is one level deeper than the cpus entry above, same as the PaaSTA variables listed underneath the env entry below.

Reply:
Please don't. This is a top level key and not some sort of sub-key of `cpus`. Env is different because that is just a list of stuff, those are not configuration options for the file.

Reply:
Fair.